array_go118.go

```diff
@@ -122,3 +126,27 @@ func (os objectValues[T, P]) MarshalLogArray(arr zapcore.ArrayEncoder) error {
 	}
 	return nil
 }
+
+// Stringers constructs a field with the given key, holding a list of the
+// output provided by the value's String method
+//
+// Given an object that implements String on the value receiver, you
```

Comment:
Same caveat as Objects applies here. We should add a comment about it:

```
// Note that these objects must implement fmt.Stringer directly.
// That is, if you're trying to marshal a []Request, the String method
// must be declared on the Request type, not its pointer (*Request).
```

In a future change, we can add an ObjectValues equivalent if necessary that will let the String method be on `*Request` for a `[]Request`.